examples/expert_gate_perf.py

```diff
@@ -0,0 +1,466 @@
+import os
+import torch
```

Comment:
Can you remove this file and add it to [continual-learning-baselines](github.com/continualAI/continual-learning-baselines) instead? we are moving the experiments there.